src/_data/service_lines_tabber.yml

```
@@ -0,0 +1,125 @@
+en:
+  what: What
+  how: How
+
+  software-modernisation:
+    tab-one-title: Sustainable Change
+    tab-one-intro: Business can move only as fast as we can deliver quality software.
+    tab-one-list-item-one: Reduce lead time
+    tab-one-list-item-two: Reduce cost of change
+    tab-one-list-item-three: Frequent and safer releases to production
+    tab-one-list-item-four: Reduce technical debt
+    tab-one-list-item-five: Reduce defect rate
```

Either now or part of the clean-up task, what do you guys think to doing something like this?

@mattgraygithub @Dan-Bird 

```suggestion
    tab-names:
      - Reduce lead time
      - Reduce cost of change
      - Frequent and safer releases to production
      - Reduce technical debt
      - Reduce defect rate
```

I'm not 💯 on the syntax but using a YML list It would gives us more flexibility.
For example, `tab-one` here has a longer list than `tab-two`.

---

I'd be keen to speak about the intentions here. I'm not quite sure I understand the use. 

This could be done as part of the cleanup though.